editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx

```diff
@@ -15,7 +14,10 @@ import CanvasActions from '../../canvas-actions'
 import { CanvasControlsContainerID } from '../new-canvas-controls'
 
 describe('Select Mode Selection', () => {
-  beforeAll(setElectronWindow)
+  // TODO find types for Mocha `before`
```

Comment:
@enidemi I guess we should fix this before merging!

Reply:
fixed